scripts/cherry-pick

```diff
@@ -92,35 +101,32 @@ run_quilt() {
         quilt --quiltrc - "$@"
 }
 
-main() {
-    local short_opts="ho:v"
-    local long_opts="help,verbose"
-    local getopt_out=""
-    getopt_out=$(getopt --name "${0##*/}" \
-        --options "${short_opts}" --long "${long_opts}" -- "$@") &&
-        eval set -- "${getopt_out}" ||
-        { bad_Usage; return; }
-
-    local cur="" next=""
-
-    while [ $# -ne 0 ]; do
-        cur="$1"; next="$2";
-        case "$cur" in
-            -h|--help) Usage ; exit 0;;
-            -v|--verbose) VERBOSITY=$((${VERBOSITY}+1));;
-            --) shift; break;;
-        esac
-        shift;
-    done
-
-	[ -n "$TEMP_D" ] ||
-        TEMP_D=$(mktemp -d "${TMPDIR:-/tmp}/${0##*/}.XXXXXX") ||
-		{ error "failed to make tempdir"; return 1; }
-	trap cleanup EXIT
+NEW_VERSION=""
 
-    [ $# -gt 0 ] || { bad_Usage "must provide commit-ish"; return; }
+drop_cherry_pick_from_daily_branch() {
+    local local_cpick_commit="$1" daily_remote="$2"
+    cur_br=$(git rev-parse --abbrev-ref HEAD) ||
+        { error "failed to get current branch name"; return 1; }
```

Comment:
should we use fail() which already exits 1?  the cleanup handler traps on exit anyhow.

Reply:
we use gitcmd function now to handle any git comand failures